curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md

```diff
@@ -16,7 +16,7 @@ In the previous step, you nested the `h2` element, comment, and `p` element with
 </ul>
 ```
 
-Add two spaces in front of the three child elements of `main` so your HTML is more readable.
+Add two spaces in front of the `h1`, comment, and `p` elements so your HTML is more readable.
```

Comment:
```suggestion
Add two more spaces in front of the h1, comment, and p elements so that they each begin six spaces over from the start of the line in order to make your HTML more readable.
```

Reply:
The `h2`, comment and `p` are not at the beginning of the line, they come immediately under the `main` tag. If we should consider the six spaces as mentioned before, then we are assuming that they would hit the back space until they were at the beginning of the line.

Reply:
> then we are assuming that they would hit the back space until they were at the beginning of the line.

I am sorry, I have to ask about an example because I am convinced

Reply:
> 
> I am sorry, I have to ask about an example because I am confused

If I'm not mistaken the code block was initially like this,
```
    <main>
    <h2>Cat Photos</h2>
    <!-- TODO: Add link to cat photos -->
    <p>Click here to view more cat photos.</p>
    </main>
```
Since the `h2` is directly beneath the `main`, won't it be easier to ask the camper to add two more spaces so that the `h2`, comment and `p` tag from the start of the `main` tag to make their HTML more readable? It would still be six spaces from the start of the line.

Reply:
If I did not misunderstand, you prefer changing this.

> Your comment should be below the `h2` element and start 6 spaces over from the start of the line 

To something like this 

> Your comment should be below the `h2` element and indented by two more space from the `main`

I do like it but, the test which test for the number of spaces need to change from
```js
assert(
  code
    .toLowerCase()
    .match(/<\/h2>\n\s{6}<!--\s*todo:\s+add\s+link\s+to\s+cat\s+photos\s*-->/)
);
```


Because it's accounting for the number of spaces

Reply:
Ah, I see. So the above code is counting 6 spaces from the beginning of the line then? If yes, can it be changed to fit the solution I proposed?

Reply:
Of course, but I don't know how you may have to ask https://chat.freecodecamp.org/channel/contributors , or open a topic in a the forum https://forum.freecodecamp.org/

Reply:
@Njong392 How is the progress going?